content/main/docs/installation/requirements.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Requirements"
+date: 2024-11-25
+---
+
+## Operators
+The orchestrator runtime/deployment is made of two main parts: `OpenShift Serverless Logic operator` and `RHDH operator`
+
+### OpenShift Serverless Logic operator requirements
+OpenShift Serverless Logic operator resource requirements are described [OpenShift Serverless Logic Installation Requirements](https://openshift-knative.github.io/docs/docs/latest/serverless-logic/getting-started/preparing-environment.html#proc-minimal-local-environment-setup). This is mainly for local environment settings.  
+The operator deploys a Data Index service and a Jobs service.
+These are the recommended minimum resource requirements for their pods:  
+`Data Index pod`:
+```yaml
+resources:
+      limits:
+        cpu: 500m
+        memory: 1Gi
+      requests:
+        cpu: 250m
+        memory: 64Mi
+```
+`Jobs pod:`
+```yaml
+resources:
+      limits:
+        cpu: 200m
+        memory: 1Gi
+      requests:
+        cpu: 100m
+        memory: 1Gi
+```
+The resources for these pods are controlled by a CR of type SonataFlowPlatform. There is one such CR in the sonataflow-infra namespace.
+
+### RHDH operator requirements
+The requirements for RHDH operator and its components are described [here](https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html/getting_started_with_red_hat_developer_hub/ref-rhdh-sizing_rhdh-getting-started#ref-rhdh-sizing_rhdh-getting-started)
```

Comment:
I'm getting 404 not found

Reply:
I'm not sure what is wrong this that link. I can be shown in the results of [https://docs.redhat.com/search/?p=1&rows=10&q=sizing+requirements](https://docs.redhat.com/search/?p=1&rows=10&q=sizing+requirements) when when clicked, it gave the 404 :-/

Reply:
raised this with RHDH team.